File: 1-two-sum/1-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> m = new HashMap<>();
        int[] res = new int[2];
        for(int i=0;i<nums.length;i++){
            if(m.containsKey(target-nums[i])){
                res[0] =m.get(target-nums[i]);
                res[1] = i;
            }
            else{
                m.put(nums[i], i);
            }
        }
        return res;
    }
}

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        res = []
        m = {}
        for i, num in enumerate(nums):
            if target-num in m:
                return [i, m[target-num]]
            else:
                m[num] = i
            
        